### PR TITLE
fix mpiexec options for hybrid option

### PR DESCRIPTION
**Pull request by Caylo**
we probably need to test this better in mympirun itself

### CODE REVIEW

**Comment by boegel** [comment]
@Caylo +1 on catching this in the mympirun unit tests...

File: test/mpi.py [hunk 1]
```diff
@@ -340,6 +340,28 @@ def test_get_universe_ncpus(self):
             universe_ppn = inst.get_universe_ncpus()
             self.assertEqual(universe_ppn, options[opt])
 
+    def test_get_hybrid_ncpus(self):
+        inst = getinstance(mpim.MPI, Local, MympirunOption())
```
Comment (boegel) [inline test/mpi.py]: @Caylo please include a docstring to clarify what you're testing exactly

File: test/mpi.py [hunk 2]
```diff
@@ -340,6 +340,28 @@ def test_get_universe_ncpus(self):
...
+        inst.nodes = [
```
Comment (boegel) [inline test/mpi.py]: maybe collapse to this (which is more clear as well):

```python
inst.nodes = ['node1'] * 4 + ['node2'] * 4 
```

File: test/mpi.py [hunk 3]
```diff
@@ -340,6 +340,28 @@ def test_get_universe_ncpus(self):
...
+            'node1',
+            'node1',
+            'node1',
+            'node1',
+            'node2',
+            'node2',
+            'node2',
+            'node2',
+        ]
+        options = range(1,5)
```
Comment (boegel) [inline test/mpi.py]: why this range specifically?

Comment (Caylo) [inline test/mpi.py]: from 1 to the ppn in the fake nodefile

**Comment by boegel** [comment]
lgtm, thanks @Caylo!

**Pull request merged**